test/uk/gov/hmrc/economiccrimelevyaccount/connectors/IntegrationFrameworkConnectorSpec.scala

```diff
@@ -16,70 +16,40 @@
 
 package uk.gov.hmrc.economiccrimelevyaccount.connectors
 
-import org.mockito.ArgumentMatchers
 import org.mockito.ArgumentMatchers.any
-import play.api.http.HeaderNames
 import play.api.libs.json.Json
 import uk.gov.hmrc.economiccrimelevyaccount.base.SpecBase
-import uk.gov.hmrc.economiccrimelevyaccount.models.CustomHeaderNames
-import uk.gov.hmrc.economiccrimelevyaccount.models.integrationframework.FinancialDataResponse
-import uk.gov.hmrc.economiccrimelevyaccount.utils.CorrelationIdGenerator
-import uk.gov.hmrc.http.{HttpClient, HttpResponse}
+import uk.gov.hmrc.economiccrimelevyaccount.models.EclReference
+import uk.gov.hmrc.economiccrimelevyaccount.models.integrationframework.FinancialData
+import uk.gov.hmrc.http.HttpResponse
 import uk.gov.hmrc.economiccrimelevyaccount.generators.CachedArbitraries._
+import uk.gov.hmrc.http.client.{HttpClientV2, RequestBuilder}
 
 import scala.concurrent.Future
 
 class IntegrationFrameworkConnectorSpec extends SpecBase {
```

Comment:
Retries should be confirmed in this spec for the IF connector.

See example:

https://github.com/hmrc/economic-crime-levy-registration/blob/944f6bd1e935ec0ec1883d5684d6c4e4e760e06f/test/uk/gov/hmrc/economiccrimelevyregistration/connectors/DmsConnectorSpec.scala#L81